acceptance/tests/custom_facts/custom_fact_with_10001_weight_overrides_external_fact.rb

```diff
@@ -0,0 +1,31 @@
+# This test verifies that a custom fact with a weight of 10001 overrides an external fact
```

Comment:
This comment is repetitive; the `test_name` expression on the next line says the same thing.